### PR TITLE
Fix broken example code

### PR DESCRIPTION
**Pull request by robinrodricks**

### CODE REVIEW

File: Examples/Headless/GettingStarted/Program.cs [hunk 1]
```diff
@@ -14,13 +14,15 @@ static void Main()
...
+			AppCoreMethods.ulEnablePlatformFileSystem(path);
 
 			// Create Renderer
-			Renderer renderer = ULPlatform.CreateRenderer();
+			var cfg = new ULConfig();
+			Renderer renderer = ULPlatform.CreateRenderer(cfg);
 
 			// Create View
-			View view = renderer.CreateView(1980, 1024);
+			View view = renderer.CreateView(800, 600);
```
Comment (SupinePandora43) [inline Examples/Headless/GettingStarted/Program.cs:25]: What's wrong with it?

File: Examples/Headless/GettingStarted/Program.cs [hunk 2]
```diff
@@ -14,13 +14,15 @@ static void Main()
...
 
 			// Set filesystem (Ultralight requires "resources/icudt67l.dat")
-			AppCoreMethods.ulEnablePlatformFileSystem(Path.GetDirectoryName(typeof(Program).Assembly.Location));
+			var path = Path.GetDirectoryName(typeof(Program).Assembly.Location);
+			AppCoreMethods.ulEnablePlatformFileSystem(path);
 
 			// Create Renderer
-			Renderer renderer = ULPlatform.CreateRenderer();
+			var cfg = new ULConfig();
+			Renderer renderer = ULPlatform.CreateRenderer(cfg);
```
Comment (SupinePandora43) [inline Examples/Headless/GettingStarted/Program.cs:22]: That is... Api design *that is not implemented yet* 😅

File: Examples/Headless/GettingStarted/Program.cs [hunk 3]
```diff
@@ -50,11 +52,8 @@ static void Main()
 			// Get Bitmap
 			ULBitmap bitmap = surface.Bitmap;
 
-			// Swap Red and Blue channels
-			bitmap.SwapRedBlueChannels();
-
 			// Save bitmap to png file
-			bitmap.WritePng("./ultralight.png");
+			bitmap.WritePng(path + "/OUTPUT.png");
```
Comment (SupinePandora43) [inline Examples/Headless/GettingStarted/Program.cs:56]: Consider using `Path.Combine`

**Review by SupinePandora43 — CHANGES_REQUESTED** [review]

File: Examples/Headless/GettingStarted/Program.cs (hunk 2 above)
Comment (robinrodricks) [reply Examples/Headless/GettingStarted/Program.cs:22]: What? I have tested it offline and it works, no idea what you mean.

**Comment by robinrodricks** [comment]
Are you going to merge this?

**Pull request merged**